### PR TITLE
Filter input minimum 3 characters

### PR DESCRIPTION
**Pull request by Paajt**
https://github.com/user-attachments/assets/ee7dcee5-b857-44b1-8254-0660e7ad984d

**Input needs 3 characters minimum to start filtering challenges titles and descriptions.**

**What it should do:**

- When input has 3 characters or more, the filter function starts filtering challenges titles and descriptions.
- If input has less than 3 characters, do nothing.

### CODE REVIEW

File: challenges.html [hunk 1]
```diff
@@ -122,7 +122,7 @@ <h1 class="ourChallengesHeader">Our challenges</h1>
         <p>
           Or type to search for keyword
         </p>
-        <input type="text" id="userInput" placeholder="Start typing to filter">
+        <input type="text" id="userInput" placeholder="Minimum 3 characters">
       </div>
```
Comment (Emilyempa) [inline challenges.html:126]: Nice to use placeholder to make user aware of how filter works

**Review by itsJoel59 — APPROVED** [review]
Good job, and yeah I agree - smart with the placeholder text, ok to merge!

**Review by alfredbrannare — APPROVED** [review]
Looks good to me! Nice update on the placeholder as well!

**Review by NordicNomadicLife — APPROVED** [review]
nicely done ok to merge from me

**Pull request merged**